client/coral-admin/src/containers/Community/Community.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import I18n from 'coral-framework/i18n/i18n';
+import translations from '../../translations';
+import {Grid, Cell} from 'react-mdl';
+
+import styles from './Community.css';
+import Table from './Table';
+import Loading from './Loading';
+import NoResults from './NoResults';
+import Pager from './Pager';
+
+const Community = ({searchValue, onKeyDownHandler, onChangeHandler, commenters, isFetching, onHeaderClickHandler, page, totalPages, onNewPageHandler}) => (
+  <Grid>
+    <Cell col={4}>
+      <form action="">
+        <div className={`mdl-textfield ${styles.searchBox}`}>
+          <label className="mdl-button mdl-js-button mdl-button--icon" htmlFor="commenters-search">
+            <i className="material-icons">search</i>
+          </label>
+          <div className="">
+            <input
+              id="commenters-search"
+              className={`mdl-textfield__input ${styles.searchInput}`}
+              type="text"
+              value={searchValue}
+              onKeyDown={onKeyDownHandler}
+              onChange={onChangeHandler}
+            />
+          </div>
+        </div>
+      </form>
+    </Cell>
+    <Cell col={8}>
+      {
+        (isFetching)
+          ?
+          <Loading />
+          :
+          (
+            (commenters.length)
```

Comment:
If we can extract the second part of the ternary and make it a stateless component will make things much more readable but it's not mandatory